lib/schema.js

```diff
@@ -21,7 +21,8 @@ const Joi = require('joi')
 const defaults = {
   hostname: 'localhost',
   port: 8125,
-  transmit: true
+  transmit: true,
+  lynxOptions: { tcp: false }
```

Comment:
This `lynxOptions` is passed to `lynx` on [line 35 of `index.js`](https://github.com/ExpediaGroup/service-client-statsd/pull/35/files#diff-92bbac9a308cd5fcf9db165841f2d90ce981baddcb2b1e26cfff170929af3bd1R35), which does not take a `tcp` option.
I think you should instead surface this `tcp` option outside of `lynxOptions` as a top-level option.